commcare_connect/program/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+from commcare_connect.opportunity.models import DeliveryType
+from commcare_connect.utils.db import BaseModel, slugify_uniquely
+
+
+class Program(BaseModel):
+    name = models.CharField(max_length=255)
+    slug = models.SlugField(max_length=255, unique=True)
+    description = models.CharField()
+    delivery_type = models.ForeignKey(DeliveryType, on_delete=models.PROTECT)
```

Comment:
Don't we need `organization` on this?

Reply:
We do it is added in the Managed Opportunity Creation PR.